manage_sections/views.py

```diff
@@ -412,8 +421,8 @@ def add_to_section(request):
     })
 
 
-@login_required
-@lti_permission_required('manage_sections')
+# @login_required
+# @lti_permission_required('manage_sections')
```

Comment:
There are a handful of @login_required and @lti_permission_required decorators commented out